### ISSUE TITLE
Using dot in the property name

### ISSUE THREAD

**Issue by kazlauskis** [crswll/tailwindcss-theme-swapper#57]
Thanks for the plugin. Any chance we can enable dots in the property names, like this:

```JS
{
  theme: {
      margin: {
          4.5: '18px',
        }
  }
}
```

This currently renders `margin: var(--margin-4.5)` where the variable is empty. I can set up a repo to reproduce this, but if you copy it over to the config, it will be present in the current plugin's [playground](https://play.tailwindcss.com/Gt21fePNvv).

**Comment by crswll** [comment]
Fixed in #58! Thanks!

**Comment by kazlauskis** [comment]
Thanks! ⭐️
